Imagine/Cache/Resolver/PsrCacheResolver.php

```diff
@@ -209,6 +209,10 @@ private function generateIndexKey($cacheKey)
      */
     private function sanitizeCacheKeyPart($cacheKeyPart)
     {
+        if (null === $cacheKeyPart) {
+            return '';
+        }
+
```

Comment:
On merging up this will cause conflict I guess because of the refactoring in `3.x`, just to point it out.

Reply:
it actually did not conflict, but tried to add the check - which became pointless as we have the parameter declaration. i guess phpstan would have noticed as well.